client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -1,94 +1,239 @@
 /** Workflow view */
-define( [ 'utils/utils', 'mvc/ui/ui-misc' ], function( Utils, Ui ) {
+define( [ "libs/toastr", "mvc/tag", "mvc/workflow/workflow-model"  ], function( mod_toastr, TAGS, WORKFLOWS ) {
+
+    /** View of the individual workflows */
+    var WorkflowItemView = Backbone.View.extend({
+        tagName: 'tr', // name of (orphan) root tag in this.el
+        initialize: function(){
+            _.bindAll(this, 'render', '_rowTemplate', 'renderTagEditor', '_templateActions', 'removeWorkflow', 'copyWorkflow'); // every function that uses 'this' as the current object should be in here
+            mod_toastr.options.timeOut = 1500;
+            mod_toastr.options.positionClass = "toast-top-left";
```

Comment:
Could we please stick to top-right? I think all other implementations show notifications there.

Reply:
Sure, np.